docs/discover/search-sessions.asciidoc

```diff
@@ -14,10 +14,7 @@ and when your session is complete, view and manage it in *Stack Management*.
 [role="screenshot"]
 image::images/search-session.png[Search Session indicator displaying the current state of the search, which you can click to stop or save a running Search Session ]
 
-Search sessions are <<search-session-settings-kb,enabled by default>>. Saving
-a search session is only available when
```

Comment:
We removed auto-refresh limitation: https://github.com/elastic/kibana/pull/96539